internal/util/k8sutil/api.go

```diff
@@ -136,6 +139,9 @@ func DefinitionsForV1beta1CustomResourceDefinitions(crds ...apiextv1beta1.Custom
 			})
 		}
 		for _, ver := range crd.Spec.Versions {
+			if !ver.Served {
+				continue
```

Comment:
Should add a helpful debug log
```suggestion
				log.Debugf("Not adding unserved CRD %q version %q to set of owned keys", crd.GetName(), ver.Name)
				continue
```